MAUI/MauiAppBasic/Platforms/MacCatalyst/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using ObjCRuntime;
+using UIKit;
+
+namespace MauiAppBasic

```

Comment:
We discussed that MacCatalyst support is not present. Or did you get it working with system webview?

Reply:
Added code that throws PlatformNotSupportedException